### PR TITLE
Clean up cmd to use the TableRenderer interface

### PR DESCRIPTION
**Pull request by jkongie**
This PR updates the commands which use `web/presenters` to use a wrapper Presenter which implements the `TableRenderer`  interface. This reduces the large switch statement in `renderer.go` with the eventual goal of replacing the generic `interface{}` argument from the `Render` function with `TableRenderer`

As we move the rest of the presenters over in the future, ensure that we follow the same pattern

### CODE REVIEW

File: core/cmd/eth_keys_commands.go [hunk 1]
```diff
@@ -43,8 +96,7 @@ func (cli *Client) CreateETHKey(c *cli.Context) (err error) {
 		}
 	}()
 
-	var keys presenters.ETHKeyResource
```
Comment (samsondav) [inline core/cmd/eth_keys_commands.go:46]: How did this ever work?

File: core/web/presenters/eth_key.go [hunk 2]
```diff
@@ -5,22 +5,21 @@ import (
...
-	DeletedAt   gorm.DeletedAt `json:"deletedAt"`
+	Address     string       `json:"address"`
+	EthBalance  *assets.Eth  `json:"ethBalance"`
+	LinkBalance *assets.Link `json:"linkBalance"`
+	NextNonce   int64        `json:"nextNonce"`
+	LastUsed    *time.Time   `json:"lastUsed"`
+	IsFunding   bool         `json:"isFunding"`
+	CreatedAt   time.Time    `json:"createdAt"`
+	UpdatedAt   time.Time    `json:"updatedAt"`
+	DeletedAt   *time.Time   `json:"deletedAt"`
```
Comment (samsondav) [inline core/web/presenters/eth_key.go:22]: Why this change?

File: core/web/presenters/eth_key.go [hunk 3]
```diff
@@ -49,7 +48,10 @@ func NewETHKeyResource(k models.Key, opts ...NewETHKeyOption) (*ETHKeyResource,
 		IsFunding:   k.IsFunding,
 		CreatedAt:   k.CreatedAt,
 		UpdatedAt:   k.UpdatedAt,
-		DeletedAt:   k.DeletedAt,
+	}
+
+	if k.DeletedAt.Valid {
```
Comment (samsondav) [inline core/web/presenters/eth_key.go:53]: Is this still relevant? Calling Valid would make more sense on a `null.Time`

**Review by samsondav — APPROVED** [review]
Looks good overall, the changed commands probably will need A/T

File: core/cmd/eth_keys_commands.go (hunk 1 above)
Comment (jkongie) [reply core/cmd/eth_keys_commands.go:46]: The model had json tags on it

File: core/web/presenters/eth_key.go (hunk 2 above)
Comment (jkongie) [reply core/web/presenters/eth_key.go:22]: I don't want the presenters to have a dependency on `gorm`. It's a DB level concern so it shouldn't leak to the presentational layer

File: core/web/presenters/eth_key.go (hunk 3 above)
Comment (jkongie) [reply core/web/presenters/eth_key.go:53]: It's not really relevant but I didn't want to make any changes to the CLI output

**Comment by jkongie** [comment]
Fixed some merge conflicts. Ready for re-review

File: core/web/presenters/eth_key.go (hunk 3 above)
Comment (samsondav) [reply core/web/presenters/eth_key.go:53]: Oops my mistake I thought this was a `ETHKeyResource` but its a `models.Key`. This is fine.

**Review by samsondav — APPROVED** [review]
